README.md

```diff
@@ -1,102 +1,170 @@
-# Puppet module: teagent
+# Puppet module: te_agent
 
 This is a Puppet module for the ThousandEyes Enterprise Agent.  
 
 
 Platform
 --------
 - Ubuntu 14.04(trusty) and 16.04 (xenial)
-- CentOS >=6.3
-- Red Hat >=6.3
+- CentOS/RedHat >=6.3
 
+**Note:** This module needs a Puppet version greater or equal to 4 .
```

Comment:
Are we sure it runs with puppet4?

Anyway, this should say that this needs Puppet 4 or Puppet 5, we don't know if this will be compatible with future Puppet 6 once it is released